### ISSUE TITLE
[13.0] sale_order_line_number

### ISSUE THREAD

**Issue by rshahid93** [ingadhoc/sale#363]
When trying to edit a sales order after it has been saved once, an error appears "sale.order.line(<NewId origin=297>,).number"

Edit inclues: Adding new sale order line (product), changing customer, changing quotation date, etc.

![image](https://user-images.githubusercontent.com/42788802/93438607-3723bc00-f8d6-11ea-90bb-a4a0a6a1aaa2.png)

Error Video: https://youtu.be/ucOUk8Bc2Lk

**Comment by jjscarafia** [comment]
Hi @rshahid93 
I guess this is v13, isn't it?
Thanks for the detailed video. I've tried to reproduce it locally but couldn't. 
Can you try to reproduce it on runbot: http://runbot3.adhoc.com.ar/runbot/repo/git-github-com-ingadhoc-sale-git-8
Are you sure you have this code on your instance? https://github.com/ingadhoc/sale/blob/13.0/sale_order_line_number/models/sale_order_line.py

**Comment by rshahid93** [comment]
@jjscarafia yes this is V13. This is so strange the way it behaves on runbot is completely different than it is behaving on my installation.I noticed on Runbot, the 'number' field stays 0 untill record is saved. On my installation it generates the serial 1,2,3,4 on creation of a new line

runbot install (0,0,0,0)
![image](https://user-images.githubusercontent.com/42788802/93505949-39146c00-f924-11ea-9be3-d143ead21e9b.png)

myinstall (1,2,3,4)
![image](https://user-images.githubusercontent.com/42788802/93506355-d1125580-f924-11ea-98c7-e9ee1e6b7a9f.png)
